src/js/tech/tech.js

```diff
@@ -339,13 +395,13 @@ class Tech extends Component {
    */
 
   textTracks() {
-    this.player_.textTracks_ = this.player_.textTracks_ || new TextTrackList();
-    return this.player_.textTracks_;
+    this.textTracks_ = this.textTracks_ || new TextTrackList();
```

Comment:
I think the tracks changes are going to be a little more complicated then just shifting things to the tech like this. @gkatsev should review this part.